src/jquery.ime.inputmethods.js

```diff
@@ -927,7 +927,7 @@
 			source: 'rules/ti/ti-geezim.js'
 		},
 		'tig-geezim': {
-			name: 'ትግረ',
+			name: 'Tigre GeezIM',
```

Comment:
I find some inconsistencies in this file regarding what should be included in the name. For some languages, we are using the autonym for some keyboard names. Do you have a strong preference for one over the other for `tig-geezim`? How should we address this?

Reply:
Thanks for the comment. You are right—it's not entirely consistent. I usually write the English name when I create the layout myself. When someone else adds something, I usually don't object to the name. In this case, however, it should have been changed for at least two reasons:
# The spelling is incorrect.
# Even if the spelling was correct, it's just the language name without a keyboard name, and we usually do write some name for the keyboard (even if there's only one keyboard for the language).

I propose that I change it in all the *-geezim layouts, but after this one is merged (simply because I don't really enjoy updating existing patches on GitHub and dealing with rebases and conflicts and all that).